components/vf-badge/vf-badge.angular.component.ts

```diff
@@ -0,0 +1,40 @@
+/*This is the  code which needs to be built as Angular library and published as npm module before it can be used in any Angular project */
+import { Component, Input, OnInit } from "@angular/core";
+
+@Component({
+  selector: "vf-badge",
+  template: `
+    <a [attr.id] = "id !== undefined ? id : null" *ngIf="badge_href !== ''" [href]="badge_href" [class]="class">{{content}}</a>
+    <span [attr.id] = "id !== undefined ? id : null" *ngIf="!badge_href || badge_href === null || badge_href === ''" [class]="class">{{content}}</span>
+  `,
+  styleUrls: []
+})
+
+export class VfBadgeAngularComponent implements OnInit {
+  /* Initialize values based on input values */
+  @Input() badge_href = '';
+  @Input() theme = '';
+  @Input() text = '';
+  @Input() style = [];
+  @Input() html = '';
```

Comment:
I'm assuming that through this input, you want to provide anyone that use the component, the option to put some HTML content inside the vf-badge. If that is the case, to display it properly you would need to do something like this: `<a [innerHTML]="content"></a>`

Reply:
Thanks @pacope92. Changes added with latest commit. Please review.